Doc/library/difflib.rst

```diff
@@ -18,7 +18,7 @@
 --------------
 
 This module provides classes and functions for comparing sequences. It
-can be used for example, for comparing files, and can produce difference
+can be used for example, for comparing files, and can produce different
 information in various formats, including HTML and context and unified
```

Comment:
```suggestion
can be used for example, for comparing files, and can produce information
about file differences in various formats, including HTML and context and unified
```

Reply:
@Mariatta that definitely sounds much better. I'll go ahead and commit suggestions. Thanks!